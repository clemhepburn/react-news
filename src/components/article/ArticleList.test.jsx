import React from 'react';
import { render } from '@testing-library/react';
import ArticleList from './ArticleList';

describe('article list', () => {
  it('should render a list of articles', () => {
    const articles = [{
      title: 'bad movie',
      author: 'Roger Ebert',
      url: 'https://www.rogerebert.com'
    }];

    const { asFragment } = render(<ArticleList articles={articles} />)
    expect(asFragment).toMatchSnapshot();
  });
});