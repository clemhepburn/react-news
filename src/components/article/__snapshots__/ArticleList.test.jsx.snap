// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`article list should render a list of articles 1`] = `[Function]`;
